gcd/src/main/java/org/spine3/server/storage/datastore/DsAggregateStorage.java

```diff
@@ -279,6 +286,22 @@ private Key keyFor(I id) {
         return key;
     }
 
+    private static class IndexExtractror<I> implements Function<Entity, I> {
```

Comment:
It's a transformer, not extractor.

Also, please have a documentation for this class.